misc/opentelemetry/mzcompose.py

```diff
@@ -18,7 +18,8 @@
         "jaeger-all-in-one",
```

Comment:
I feel like we should just call this `jaeger`! `jaeger-all-in-one` is such a mouthful.